### PR TITLE
Update about content

### PR DESCRIPTION
**Pull request by anthonydillon**
Updated all URLs, content and navigation.

### CODE REVIEW

File: templates/about/integration.html [hunk 1]
```diff
@@ -4,49 +4,98 @@
...
+Use the ‘relate’ command to integrate two operators.
+</p>
+<pre><code>
+juju relate mysql wordpress
+</code></pre>
+<p>
+In the simple case, if there is only one pair of matching integration points for those two operator charms, then this command will work. If there are multiple possible ways to relate the two operators, with multiple pairs of matching integration points, then you will need to name the points explicitly.
+</p>
+<pre><code>
+juju relate mysql wordpress:db
```
Comment (pmahnke) [inline templates/about/integration.html:84]: same, remove newlines

File: templates/about/operator-lifecycle-manager.html [hunk 2]
```diff
@@ -4,58 +4,140 @@
...
 {% block meta_copydoc %}https://docs.google.com/document/d/1B0adX3iTzNrF_RORQUWU6MaYn_E2wzvfJYUbolAY3Ok/edit{% endblock meta_copydoc %}
 
-{% block meta_description %}An operator is software which drives an application. The operator lifecycle manager (OLM) is a service which installs operators, updates them, configures them, and in the case of the Juju OLM, helps it to integrate with other operators.{% endblock %}
+{% block meta_description %}Juju is an operator lifecycle manager (OLM). A service which installs operators, updates them, configures them, and helps them integrate with other operators.{% endblock %}
 
 {% block about_content %}
 <h1>What is an Operator Lifecycle Manager?</h1>
-<p>A service that manages the install, update, configuration and integration of operators on Kubernetes or traditional machines</p>
-<p>An operator is software which drives an application. The operator lifecycle manager (OLM) is a service which installs operators, updates them, configures them, and in the case of the Juju OLM, helps it to integrate with other operators.</p>
+<p>Juju manages the install, update, configuration and integration of operators on Kubernetes or traditional machines
```
Comment (pmahnke) [inline templates/about/operator-lifecycle-manager.html:11]: p-heading--four

File: templates/about/operator-lifecycle-manager.html [hunk 3]
```diff
@@ -4,58 +4,140 @@
...
+<p>
+One key differentiator of the Juju OLM is the way it organises a large number of operators, potentially across many different clouds or clusters. Operators are in a sense extra pieces of software to manage, so having a structure and catalog of that software immediately provides a level of enterprise awareness and control of the operator estate.
+</p>
+<p>
+The Juju OLM provides <a href="https://juju.is/model-driven-operations">model-driven operations</a>. Sets of operators are organised into models, each model having the same set of access permissions and living on the same Kubernetes cluster or machine compute substrate such as a public cloud in the case of traditional applications. The ability to organise multi-cloud operations in these models, and to manage models globally, across many regions of many clouds and private infrastructure, through a single scale-out OLM, is a strength of Juju.
```
Comment (pmahnke) [inline templates/about/operator-lifecycle-manager.html:21]: make the link relative?

File: templates/about/operator-services.html [hunk 4]
```diff
@@ -4,35 +4,76 @@
...
 
 {% block about_content %}
 <h1>Operator lifecycle manager services</h1>
-<p>The Juju operator lifecycle manager (OLM) enables carrier-grade, highly available operators by providing distributed systems services that reduce the cost and complexity of operator implementation for mission-critical applications and infra.</p>
-<p>It is easy to write a simplistic operator. Launch a container, fetch a new container, and you’re done. It is challenging to write a highly available operator that handles distributed systems gremlins smoothly and scales efficiently with automated integration to other operators.</p>
-<p>The Juju OLM provides enterprise-class operator lifecycle management and it offers services to operators which make it easier for them, in turn, to be enterprise-class components. Operators for enterprise-class services may well need to be highly available, and so they become distributed systems in their own right.</p>
+<p>The Juju operator lifecycle manager (OLM) enables carrier-grade, highly available operators. It provides services that reduce the cost and complexity of operators for mission-critical apps and infra.
```
Comment (pmahnke) [inline templates/about/operator-services.html:11]: p-heading--four

File: templates/about/multi-cloud-operations.html [hunk 5]
```diff
@@ -4,63 +4,148 @@
...
+</p>
+<p>
+We talk about operators as ‘driving software’, managing the lifecycle of application installation and upgrade and configuration. But there is no reason for that software to be a local application. SAAS offerings have a lifecycle too. ‘Install the application’ maps to ‘call the API to instantiate the service’. ‘Remove the application’ maps to ‘call the API to terminate the service’. Each service has a set of things which need to be negotiated in order to integrate with it, just like an application does.
+</p>
+<p>
+So, in the <a href="https://charmhub.io/">open operator collection</a>, you will find operators for common SAAS solutions. You can install these operators into your model, you can configure them, you can integrate with them, and they behave in the model exactly as any other operator. The only difference is that they drive the APIs of the SAAS provider, rather than driving local software installation.
```
Comment (pmahnke) [inline templates/about/multi-cloud-operations.html:143]: p-link--external

File: templates/about/hosted-olm.html [hunk 6]
```diff
@@ -4,20 +4,37 @@
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1zmGR4hZyyTLuPc98XahtcM7Tlb35QYzXZ9dWRf50l8Y/edit{% endblock meta_copydoc %}
 
-{% block meta_description %}Centralised OLM for multi-cloud enterprise operations{% endblock %}
+{% block meta_description %}Run JAAS (Juju As A Service) either on your premises or hosted at jaas.ai across most of the public clouds.{% endblock %}
 
 {% block about_content %}
 <h1>Operator Lifecycle Manager SAAS</h1>
-<p class="p-heading--four">Centralised OLM for multi-cloud enterprise operations</p>
+<p>Centralised enterprise OLM for multi-cloud operations
```
Comment (pmahnke) [inline templates/about/hosted-olm.html:11]: p-heading--four

File: templates/about/hosted-olm.html [hunk 7]
```diff
@@ -4,20 +4,37 @@
...
+In larger organisations it is useful to provide a single global view of all of the models containing all the operators running across all the clouds and Kubernetes clusters. JAAS on premise provides an aggregated view across multiple Juju controllers, and serves as a single admin interface for OLM models.
+</p>
+<p>
+JAAS is provided by Canonical and enables enterprise RBAC for models on public clouds, private infrastructure, bare metal, VMware, and OpenStack. For organisations with many Juju controllers, JAAS on premise offers a central point for management, monitoring and audit.
+</p>
+<p>
+JAAS acts as a policy-enforcing proxy to underlying Juju controllers, which enables it to scale to thousands of models across as many controllers as are needed. Identity management integration with the enterprise allows for single-sign-on to the Juju service.
+</p>
+<p>
+You can try JAAS.ai, a public hosted version of the same code. Bringing JAAS on premise creates your own secure, centralized Operator Lifecycle Manager service for the entire business. For more information, please <a href="/contact-us">get in touch</a>.
```
Comment (pmahnke) [inline templates/about/hosted-olm.html:38]: /contact-us is 404ing

**Review by pmahnke — CHANGES_REQUESTED** [review]
Comments inline

File: templates/about/hosted-olm.html (hunk 7 above)
Comment (anthonydillon) [reply templates/about/hosted-olm.html:38]: @barrymcgee is on this.

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**